tools/packaging/deploy/enclave-cc-deploy.sh

```diff
@@ -25,6 +25,16 @@ function print_usage() {
 	echo "Usage: $0 [install/cleanup/reset]"
 }
 
+function create_runtimeclass() {
+	echo "Creating the runtime classes"
+	kubectl apply -f runtimeclass/enclave-cc.yaml
```

Comment:
I built the payload image and I *think* the path is not correct since `enclave-cc-deploy.sh` is under `/scripts/` and `enclave-cc.yaml` is `/runtimeclass`.